plugins/cisco_umbrella_enforcement/plugin.spec.yaml

```diff
@@ -116,68 +34,182 @@ types:
       description: The number of results
       required: true
     prev:
-      type: string
+      type: boolean
       description: If prev is false, this is the first available page of results.
         Otherwise, it will provide a query formatted to show the next set of results
       required: true
     next:
-      type: string
+      type: boolean
       description: If next is false, this is the last available page of results. Otherwise,
         it will provide a query formatted to show the next set of results
       required: true
   data:
-    ID:
-      type: integer
-      description: Unique ID number
-      required: true
     name:
       type: string
       description: Domain name
       required: true
+    id:
+      type: integer
+      description: Unique ID number
+      required: true
     lastSeenAt:
-      type: string
+      type: integer
       title: Last seen malware
       description: Unix timestamp last seen
       required: true
+    meta:
+      type: meta
+      description: ejejeje
```

Comment:
what does ejejeje mean?

Reply:
good spot i think i just wrote this as filler and forgot to replace it